brontology/graph/iterable/graph.py

```diff
@@ -0,0 +1,91 @@
+# noinspection PyUnresolvedReferences
+from typing import Union, Optional, TypeVar, Iterable, Generic, Type
+
+from brontology.graph.base.node import NO_CONTENT
+from brontology.graph.iterable.node import IterableNode, IterableLink, C
+
+L = TypeVar("L", bound=IterableLink)
+N = TypeVar("N", bound=IterableNode)
+M = TypeVar("M")
+
+
+class IterableGraph(Generic[N, L, M]):
```

Comment:
Hier findest du den funktionalen Teil des Entity Graphen.

Dieser `IterableGraph` funktioniert komplett losgelöst vom Typ seines Inhalts, zum Beispiel kannst du in den Tests sehen, dass man mit ihm auch einen Graphen erstellen kann, der Nodes und Relationen pflegt, die Zahlen enthalten.